cypress-tests/cypress/e2e/configs/Payment/Commons.js

```diff
@@ -1612,7 +1570,7 @@ export const connectorDetails = {
     },
   },
   return_url_variations: {
-    "return_url_too_long": getCustomExchange({
+    return_url_too_long: getCustomExchange({
       Request: {
         customer_id: "customer_1234567890",
         return_url: "http://example.com/" + "a".repeat(237),
```

Comment:
if not asking for much, can [this](https://github.com/juspay/hyperswitch/pull/8006#discussion_r2098085648) be change be incorporated in this pr?